packages/cli/src/ceramic-cli-utils.ts

```diff
@@ -132,7 +133,7 @@ export class CeramicCliUtils {
     pubsubTopic: string,
     corsAllowedOrigins: string,
     syncOverride: string,
-    enableComposedb: string | undefined
+    disableComposedb: string | undefined
```

Comment:
```suggestion
    disableComposedb: boolean
```

Reply:
can look at how the other boolean flags are handled here

Reply:
ah, yeah that was a leftover from the previous input from CLI